### PR TITLE
Issue#325branch

### PR DESCRIPTION
**Pull request by nedmsmith**

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -374,41 +374,21 @@ An Attester creates Evidence that is conveyed to a Verifier.
...
 by applying an Appraisal Policy for Evidence to assess the trustworthiness of the Attester.
 This procedure is called the appraisal of Evidence.
-
 Subsequently, the Verifier generates Attestation Results for use by Relying Parties.
-The Appraisal Policy for Evidence
-might be obtained from an Endorser along with the Endorsements, and/or might be obtained
-via some other mechanism, such as being configured in the Verifier by the Verifier Owner.
+
+The Appraisal Policy for Evidence might be obtained from the Verifier Owner,  
+and/or is configured into the Verifier by the Verifier Owner,
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:380]: These two are the same thing.  Also this change doesn't appear to be related to issue #325

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -374,41 +374,21 @@ An Attester creates Evidence that is conveyed to a Verifier.
...
-and are then used by the appraisal policy.
-
-The actual data format and semantics of any Reference Values are specific to
-Claims and implementations.
-This architecture document does not define any
-general purpose format for Reference Values or general means for comparison.
+The Appraisal Policy for Attestation Results might be obtained from the Relying Party Owner,  
+and/or is configured into the Relying Party by the Relying Party Owner, 
+and/or is programmed into the Relying Party,
+and/or might be obtained via some other mechanism.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:392]:
```suggestion

See {{messages}} for further discussion of the conceptual messages shown in {{dataflow}}.
```

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -994,6 +974,10 @@ in order to bootstrap the sequence.
 See {{privacy-considerations}} for more discussion.
 
 # Conceptual Messages {#messages}
+The data format and semantics of conceptual messages are implementation specific.
+This architecture document does not define any
+general purpose data format for conceptual messages or a general means for comparison.
+Implementations of conceptual messages will define data formats, semantics and the means for comparison.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:981]: For comparison, my PR had:

> {{dataflow}} illustrates the flow of a conceptual messages between various roles.
> This section provides additional elaboration and implementation considerations.
> It is the responsibility of protocol specifications to define the actual data format
> and semantics of any relevant conceptual messages.

Comment (henkbirkholz) [reply draft-ietf-rats-architecture.md:979]: Leaning a tad bit towards Dave's text. Ned's is also fine. My opinion should not be used as a scale-tipping one.

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -994,6 +972,10 @@ in order to bootstrap the sequence.
...
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:978]:
```suggestion
{{dataflow}} illustrates the flow of a conceptual messages between various roles.
This section provides additional elaboration and implementation considerations.
It is the responsibility of protocol specifications to define the actual data format
and semantics of any relevant conceptual messages.
```
